### ISSUE TITLE
Screen reader doesn't announce anything from niri

### ISSUE THREAD

**Issue by YaLTeR** [niri-wm/niri#3594]
This is an accesskit_unix 0.18+ regression, not clear what to do about it yet. https://github.com/AccessKit/accesskit-c/issues/76

**Comment by YaLTeR** [comment]
I downgraded the crate for now to get the screen reader working again.

**Comment by takashialpha** [comment]
any updates on this?

**Comment by YaLTeR** [comment]
If you look at the linked issue there are no updates. In niri the version is downgraded and so works for now

**Comment by takashialpha** [comment]
@YaLTeR Actually, I think that there is an update. PR # 715 on accesskit (merged May 23, 2026) fixes the issue by watching IsEnabled instead. Confirmed to properly detect screen readers starting before or after the app. It's Merged, but not released yet.
accesskit 0.24 and accesskit_unix 0.21 (specfically when 0.21.2 gets released, will include that fix) should work fine after adapting the usage in niri.

**Comment by YaLTeR** [comment]
Upgraded in fdb6d85fc78355762bcf3cf71fe4037681a766f9. Checked and it seems to work indeed!
